### DISCUSSION TITLE
Copilot hangs forever

### DISCUSSION THREAD

**Discussion by Barben360** [community/community#35437] [Copilot Conversations]
Hello,

I have a problem with Github Copilot hanging forever on VSCode. I have a Wi-Fi connection that sometimes has some micro-failures, but it seems that Copilot is not able to recover from it. The Copilot icon working animation keeps turning and never stops and I don't get any suggestion anymore until I restart VSCode.

Is it possible to investigate it?

**Comment by iamcjwilliams** [comment]
@Barben360 Have you upgraded to the latest version? If so, can you post more information by performing the following: (View -> Output; select GitHub Copilot from the pulldown)
